libs/wingsdk/src/shared-aws/bucket.inflight.ts

```diff
@@ -16,11 +16,11 @@ import {
 } from "@aws-sdk/client-s3";
 import { BucketDeleteOptions, IBucketClient } from "../cloud";
 import { Duration, Json } from "../std";
-
+import { getSignedUrl ,S3RequestPresigner} from "@aws-sdk/s3-request-presigner";
```

Comment:
is `S3RequestPresigner` used?

Reply:
@tsuf239 I dont think so 
This is unwanted , Will remove this